applications/app/views/signup/newsletterContent.scala.html

```diff
@@ -116,6 +116,9 @@ <h3 class="signup-confirmation--success">You've subscribed!</h3>
         <div class="newsletters-category" name="comment">
             @emailListCategoryList("comment", EmailNewsletters.commentEmails)
         </div>
+        <div class="newsletters-category" name="work">
+            @emailListCategoryList("work", EmailNewsletters.workEmails)
+        </div>
```

Comment:
can you refactor this entire list into a proper list?

sort of like this

    Map(
       "comment" -> EmailNewsletters.commentEmails
       "work" -> EmailNewsletters.workEmails
       ...
    ) foreach ( newsletter => 
       <div class="newsletters-category" name="@newsletter._1">
              @emailListCategoryList(@newsletter._1, @newsletter._2)		              
       </div>
    )